<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="3" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-27T16:12:36 EDT" name="Default suite" finished-at="2023-07-27T16:13:00 EDT" duration-ms="23896">
    <groups>
    </groups>
    <test started-at="2023-07-27T16:12:36 EDT" name="Default test" finished-at="2023-07-27T16:13:00 EDT" duration-ms="23896">
      <class name="com.testNG.assignment.First_TestNG_Assignment">
        <test-method is-config="true" signature="commonFunction()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:36 EDT" name="commonFunction" finished-at="2023-07-27T16:12:42 EDT" duration-ms="5870" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonFunction -->
        <test-method signature="registerTestCase2()[pri:1, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:42 EDT" name="registerTestCase2" finished-at="2023-07-27T16:12:43 EDT" duration-ms="1478" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerTestCase2 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:43 EDT" name="tearDown" finished-at="2023-07-27T16:12:43 EDT" duration-ms="172" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="commonFunction()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:43 EDT" name="commonFunction" finished-at="2023-07-27T16:12:47 EDT" duration-ms="3565" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonFunction -->
        <test-method signature="registerTestCase1()[pri:2, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:47 EDT" name="registerTestCase1" finished-at="2023-07-27T16:12:48 EDT" duration-ms="1499" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at com.testNG.assignment.First_TestNG_Assignment.registerTestCase1(First_TestNG_Assignment.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerTestCase1 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:48 EDT" name="tearDown" finished-at="2023-07-27T16:12:48 EDT" duration-ms="122" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method signature="registerTestCase3()[pri:3, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:48 EDT" name="registerTestCase3" finished-at="2023-07-27T16:12:48 EDT" depends-on-methods="com.testNG.assignment.First_TestNG_Assignment.registerTestCase1, com.testNG.assignment.First_TestNG_Assignment.registerTestCase2" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.testNG.assignment.First_TestNG_Assignment.registerTestCase3() on instance com.testNG.assignment.First_TestNG_Assignment@2cf3d63b depends on not successfully finished methods 
[com.testNG.assignment.First_TestNG_Assignment.registerTestCase1() on instance com.testNG.assignment.First_TestNG_Assignment@2cf3d63b
com.testNG.assignment.First_TestNG_Assignment.registerTestCase2() on instance com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.testNG.assignment.First_TestNG_Assignment.registerTestCase3() on instance com.testNG.assignment.First_TestNG_Assignment@2cf3d63b depends on not successfully finished methods 
[com.testNG.assignment.First_TestNG_Assignment.registerTestCase1() on instance com.testNG.assignment.First_TestNG_Assignment@2cf3d63b
com.testNG.assignment.First_TestNG_Assignment.registerTestCase2() on instance com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registerTestCase3 -->
        <test-method is-config="true" signature="commonFunction()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:48 EDT" name="commonFunction" finished-at="2023-07-27T16:12:52 EDT" duration-ms="3690" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonFunction -->
        <test-method signature="loginTestCase1()[pri:4, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:52 EDT" name="loginTestCase1" finished-at="2023-07-27T16:12:54 EDT" duration-ms="1857" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestCase1 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:54 EDT" name="tearDown" finished-at="2023-07-27T16:12:54 EDT" duration-ms="128" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="commonFunction()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:54 EDT" name="commonFunction" finished-at="2023-07-27T16:12:58 EDT" duration-ms="3556" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- commonFunction -->
        <test-method signature="loginTestCase2()[pri:5, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:58 EDT" name="loginTestCase2" finished-at="2023-07-27T16:12:59 EDT" duration-ms="1794" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTestCase2 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.testNG.assignment.First_TestNG_Assignment@2cf3d63b]" started-at="2023-07-27T16:12:59 EDT" name="tearDown" finished-at="2023-07-27T16:13:00 EDT" duration-ms="114" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testNG.assignment.First_TestNG_Assignment -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
